tracing/src/span.rs

```diff
@@ -1162,6 +1162,11 @@ impl Span {
     ///     span was created:
     /// </pre>
     ///
+    /// <pre class="ignore" style="white-space:normal;font:inherit;">
+    ///     <strong>Note</strong>: <a href="https://docs.rs/tracing-subscriber/latest/tracing_subscriber/fmt/struct.Subscriber.html"><code>FmtSubscriber</code></a> has a bug
+    ///     where it duplicates span fiels because of its implementation. See details at:  <a href="https://github.com/tokio-rs/tracing/issues/1372">Github</a>
```

Comment:
```suggestion
    ///     where it duplicates span fields because of its implementation. See details at:  <a href="https://github.com/tokio-rs/tracing/issues/1372">Github</a>
```